packages/beacon-node/.mocharc.spec.cjs

```diff
@@ -3,6 +3,5 @@ module.exports = {
   require: ["./test/setupPreset.ts", "./test/setup.ts"],
   "node-option": ["loader=ts-node/esm"],
   timeout: 60_000,
-  // Do not run tests through workers, it's not proven to be faster than with `jobs: 2`
```

Comment:
Why does this no longer apply?

Reply:
Because I parallized the spec tests. It was not able to parallize because everything was put in one `index.test.ts` and mocha can only parallize test suits not test cases.

Reply:
That makes sense to me but the comment suggest that running tests in workers did not make them faster. I guess it was just a stale comment then

Reply:
> running tests in workers did not make them faster

because all tests were running in one test suite `index.test.ts` before. So why there was no performance improvement visible.

Reply:
I think I got it, so it meant we anyhow just parallelize 2 jobs which is not worth it (overhead of spawning workers i'd assume) but now have way more jobs, hence it becomes worth it to run them in parallel.